test/Ocelot.AcceptanceTests/DownstreamMetadataTests.cs

```diff
@@ -0,0 +1,609 @@
+﻿using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;
+using Ocelot.Metadata;
+using Ocelot.Middleware;
+using System.Globalization;
+
+namespace Ocelot.AcceptanceTests;
```

Comment:
We should not define classes in the root folder!
Each feature should have separate folder.